### ISSUE TITLE
Discover page doesn't fetch anything....

### ISSUE THREAD

**Issue by Cyb3r7ron** [adrianhajdin/project_music_player#18]
Discover page doesn't fetch anything, also lyrics page or artist details page is not working....please help any ideas on how to fix it?
![discover err](https://user-images.githubusercontent.com/45810961/204303293-6ee94d84-aa5b-4f42-9276-d22274d1d613.PNG)

**Comment by hungdev20** [comment]
Have you solved this problem yet?

**Comment by sayedulkrm** [comment]
Check your links. Spelling mistake provably

**Comment by Cyb3r7ron** [comment]
> Have you solved this problem yet?

Yes apparently, the problem was punctuation problem it had to be this ` instead I was using this ' in the parameters.
